src/pages/Home/StoreBlock.js

```diff
@@ -24,18 +24,26 @@ const StoreContainer = styled.div`
     }
   }
   @media ${device.tablet} {
+    display: flex;
```

Comment:
I'm 99% sure that since we're declaring this at the mobile media query, we don't need to re-state the display, flex-direction, and align-content again. The same rules you're setting at device.mobile should still apply here.

Reply:
@dbridenbeck its true, I removed it